src/helix-ui/styles/core/hx-modal.less

```diff
@@ -0,0 +1,49 @@
+hx-modal {
+  color: initial;
+  display: none;
+  font-family: "Roboto", sans-serif;
+  font-size: 1rem;
+  font-style: normal;
```

Comment:
With the suggested shadow dom change above, do we still need all these overrides/resets?